src/frontend/dashboard-layout.js

```diff
@@ -88,7 +88,7 @@ _.extend(DashboardLayout.prototype, {
 				else {
 					$layout.addClass('knsh-dashboard-layout-group__m-col');
 				}
-		
+				
```

Comment:
Yes, I prefer that empty lines have the same indentation as the surrounding non-empty lines. My text editor does not understand my preference and I have to keep eye on this myself and sometimes I fail.